README.md

```diff
@@ -1 +1,84 @@
-# go-borges
\ No newline at end of file
+[![GoDoc](https://godoc.org/github.com/src-d/go-borges?status.svg)](https://godoc.org/github.com/src-d/go-borges)
+[![Build Status](https://travis-ci.org/src-d/go-borges.svg)](https://travis-ci.org/src-d/go-borges)
```

Comment:
Travis links should point to `travis-ci.com` instead of `travis-ci.org`. We need to change this also in other projects like gitcollector.

Reply:
You're right, and also in the guide. Fixed